.github/workflows/issues_and_pull_requests.yml

```diff
@@ -52,6 +52,8 @@ jobs:
           days-before-pr-close: "${{ env.DAYS_BEFORE_PR_CLOSE }}"
           days-before-pr-stale: "${{ env.DAYS_BEFORE_PR_STALE }}"
           exempt-all-milestones: true
```

Comment:
tbh I will struggle myself to recognise the difference between only milestone we have (`long-term`) and label `topic/core` (which we don't want to close by bot even if it's... open for long term ;) 

OK with any approach, wondering if we need both.

Reply:
Back in the days I've used "long term ideas" milestone as a quick hack, but I don't want to use it for all issues. It's a too wide bag for everything and nothing. I would prefer to clean it up at some point.

Reply:
would you prefer keeping both or removing hacky milestone ?
[ok with both, just wondering which way to go for next PR of mine that will wait few years]

Reply:
I would use meaningful milestones instead of one "long term", just for grouping issues/PRs related to one goal. All of the milestones are excluded from Stale Bot, since all of them are part of a vision (short term or long term). `topic/core` is for single issues/PRs that are not part of any milestone. Both (milestones and labels) should be used where applicable 🙂.

Reply:
I read above as you do not want to have hacky milestone "long term" and if anything having this milestone, you would convert them to topic/core

Reply:
Yes, I already did that for most of the issues.